tests/test_file_info.py

```diff
@@ -22,16 +22,34 @@ class TestBitrate(unittest.TestCase):
 
     def test_wav(self):
         actual = file_info.bitrate(INPUT_FILE)
-        expected = 16
+        expected = 706000.0
         self.assertEqual(expected, actual)
 
     def test_aiff(self):
         actual = file_info.bitrate(INPUT_FILE2)
-        expected = 32
+        expected = 768000.0
         self.assertEqual(expected, actual)
 
     def test_empty(self):
         actual = file_info.bitrate(EMPTY_FILE)
+        expected = None
+        self.assertEqual(expected, actual)
+
+
+class TestBitdepth(unittest.TestCase):
+
+    def test_wav(self):
+        actual = file_info.bitdepth(INPUT_FILE)
+        expected = 16
+        self.assertEqual(expected, actual)
+
+    def test_aiff(self):
+        actual = file_info.bitdepth(INPUT_FILE2)
+        expected = 32
+        self.assertEqual(expected, actual)
+
+    def test_empty(self):
+        actual = file_info.bitdepth(EMPTY_FILE)
         expected = 16
         self.assertEqual(expected, actual)
```

Comment:
We should test for the "unavailable" case as well - maybe an empty file in a different format than the current one in the tests folder would trigger this?

Reply:
Yes. I don't know how to generate such a file though.
`sox -n -c 1 -b 0 empty_zerobit.wav trim 0.0 10.0` does not work, of course.
Calling `sox --i -b` on one of the existing files either results in a positive number (16, 32) or a `SoxError`

Reply:
This type of file probably exists or SoX probably wouldn't have a case handling it, but I also haven't been able to generate one. Maybe it's possible for a file format that we never use??